content/updates/productivity/3-A-more-helpful-process-tracker.md

```diff
@@ -0,0 +1,48 @@
+# More helpful process tracker
+
+## See all the ongoing jobs
+::: meta-data type="Improvement" ee-only="false" link-to-doc="../articles/monitor-jobs.html"
+:::
+
+To give you more visibility and control on all PIM jobs, we added a new permission: `View all jobs in the Process Tracker` to see all jobs of any status under the Process Tracker.
+
+You will now be able to see the ongoing or previous PIM jobs and, if needed, which users launched them. :wink:
```

Comment:
You will now be able to see all the PIM jobs that are started or have been completed and, if needed, which users launched them. :wink: